ui/src/map/layers/geometry/plan-area-layer.ts

```diff
@@ -1,6 +1,6 @@
 import mapStyles from 'map/map.module.scss';
 import Feature from 'ol/Feature';
-import { Polygon } from 'ol/geom';
+import { Polygon as OlPolygon } from 'ol/geom';
```

Comment:
Näitä muutoksia on jonkin verran. Tein meille uuden Geoviitteen sisäistä polygonia kuvaavan `Polygon`-tyypin ja renamesin nämä alta `OlPolygon`:eiksi